internal/contracts/contracts.go

```diff
@@ -273,5 +275,30 @@ func (mc *JSONContract) Unmarshal() (Contract, error) {
 		mc.Value,
 		mc.StateNonce,
 	}
-	return c, nil
+	return c, nil 
+	
 }
+
+
+
+func GenerateRandomContract() (*Contract){
+	min := 1
+	maxVer := 65535
+	b := make([]byte, 32)
+	mrand.Read(b)
+	genRecipPubKeyHash := b 
```

Comment:
Can you make this the result of the hash of a random number? 

i.e. `hashing.New([]byte{ someRandomNumber })`

Reply:
does it matter how long the number inside the byte is, as in does it matter if it is a random Uint16 or Uint64

Reply:
I thought an array of bytes was multiple values, not just one singular number

Reply:
> does it matter how long the number inside the byte is, as in does it matter if it is a random Uint16 or Uint64

I don't think it matters. Choose whichever is easier for you

Reply:
is it possible to have an array of bytes with just a single number

Reply:
i thought it was an array with multiple numbers